.github/ISSUE_TEMPLATE/GC-Articles-Publishing

```diff
@@ -0,0 +1,57 @@
+---
+name: GC Articles Publishing User Story Template
+about: Template for creating new User Stories for Notify backlog
+title: 'Give your user story a simple, clear title'
+labels: ''
+assignees: ''
+
+---
+## Description
+As a (user), I need to be able to do (X) so that I can achieve (Y) outcome. 
+
+WHY are we building?
+WHAT are we building?
+VALUE created by our solution
+
+## Documentation and Artifacts
+Good docs, figma mockups, ADRs, screenshots etc.
+
+## Acceptance Criteria
+Given some context, when (X) action occurs, then (Y) outcome is achieved 
+
+- [ ] Cypress UI tests if needed.
+- [ ] Generate appropriate log messages so that executions of this feature can be tracked
+- [ ] Can misuse of this feature cause harm? If yes, create an alert
+- [ ] Update the status of related findings, insights, and hypotheses on the Research Airtable
+- [ ] Once change/fix/feature is implemented, link relevant Airtable records to design artifacts (Figma)
```

Comment:
If this is strictly for GCA publishing, we could probably remove some stuff in here. I don't think we need:
- alerts
- log messages
- Are we still using aritable? Or more likely, who is updating airtable?